client/keys.go

```diff
@@ -67,14 +69,32 @@ func EndorsementKeyFromNvIndex(rw io.ReadWriter, idx uint32) (*Key, error) {
 // function will only work on a GCE VM. Unlike AttestationKeyRSA, this key uses
 // the Endorsement Hierarchy and its template loaded from GceAKTemplateNVIndexRSA.
 func GceAttestationKeyRSA(rw io.ReadWriter) (*Key, error) {
-	return EndorsementKeyFromNvIndex(rw, GceAKTemplateNVIndexRSA)
+	akRsa, err := EndorsementKeyFromNvIndex(rw, GceAKTemplateNVIndexRSA)
+	if err != nil {
+		return nil, err
+	}
+	akCert, err := getCertificateFromNvram(rw, GceAKCertNVIndexRSA)
+	if err != nil {
+		return nil, fmt.Errorf("failed to fetch GCE RSA AKCert: %v", err)
+	}
+	akRsa.cert = akCert
+	return akRsa, nil
 }
 
 // GceAttestationKeyECC generates and loads the GCE ECC AK. Note that this
 // function will only work on a GCE VM. Unlike AttestationKeyECC, this key uses
 // the Endorsement Hierarchy and its template loaded from GceAKTemplateNVIndexECC.
 func GceAttestationKeyECC(rw io.ReadWriter) (*Key, error) {
-	return EndorsementKeyFromNvIndex(rw, GceAKTemplateNVIndexECC)
+	akEcc, err := EndorsementKeyFromNvIndex(rw, GceAKTemplateNVIndexECC)
+	if err != nil {
+		return nil, err
+	}
+	akCert, err := getCertificateFromNvram(rw, GceAKCertNVIndexECC)
```

Comment:
Here (and for any other GCE keys), we shoudn't fail if the cert isn't present, just don't set it.